docs/content/concepts/assets/software-defined-assets.mdx

```diff
@@ -198,9 +198,7 @@ In this example, Dagster doesn’t need to load data from `upstream_asset` to su
 
 ### Graph-backed assets
 
-[Basic software-defined assets](#a-basic-software-defined-asset) can only produce one data artifact. If generating an asset involves multiple discrete computations, you can use graph-backed assets by separating each computation into an op and building a graph to combine your computations. This way, each discrete computation can be reused in other assets and jobs.
-
-This approach to asset creation allows you to define a graph of ops which can produce one or multiple assets. **Note**: To use graphs to create an asset, the graph **must return an output**.
```

Comment:
I liked this phrase - it made sense to me that graph-backed assets are assets that generated from graphs.

Reply:
I had taken it out because it felt redundant with this sentence: "If generating an asset involves multiple discrete computations, you can use graph-backed assets by separating each computation into an op and building a graph to combine your computations." Is there information in the latter sentence that's missing in the former?

Reply:
After the discussion we had on the other comment, I think this comment was more related to the graph to asset path, as opposed to separating one asset to multiple ops. The sentence you quoted is sufficient and the other discussion should be able to address my confusion.